includes/configs/strings.php

```diff
@@ -27,6 +27,7 @@
 	'image'                                     => esc_html__( 'Image', 'hivepress' ),
 	'images'                                    => esc_html__( 'Images', 'hivepress' ),
 	'select_images'                             => esc_html__( 'Select Images', 'hivepress' ),
+	'default'                                   => esc_html__( 'Default Value', 'hivepress' ),
```

Comment:
It's ok to remove this string, we keep this config as minimal as possible and it's needed only if there's an external extension which uses the string, or if the string is business-specific (e.g. "listing") and themes need to override the strings from the outside. Also, if you need to add a string, make the key as similar as possible to the string itself